MOBILE_CHECKOUT_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,190 @@
+# PayPal Mobile Checkout SDK: Migration Guide
+
+This guide outlines how to update your integration from using the soon-to-be-deprecated [PayPal Mobile Checkout SDK](https://developer.paypal.com/limited-release/paypal-mobile-checkout/) to the new PayPal Mobile [iOS SDK](https://github.com/paypal/iOS-SDK).
+
+## Pre-Requisites
+In order to use this migration guide, you must:
+
+1. Have a server-side integration with the [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2/). Please update to Orders v2 if you're on [Payments V1](https://developer.paypal.com/docs/api/payments/v1/) or [NVP/SOAP](https://developer.paypal.com/api/nvp-soap/).
+1. Enable your server to [fetch an Access Token](https://developer.paypal.com/reference/get-an-access-token/).
+1. Enable your server to create an [Order ID](https://developer.paypal.com/docs/api/orders/v2/).
+1. Enable your server to [PATCH](https://developer.paypal.com/docs/api/orders/v2/#orders_patch) an order.
+    * _Note:_ This is **only required** if you create your order ID with [`shipping_preference`](https://developer.paypal.com/docs/api/orders/v2/#definition-order_application_context) = `GET_FROM_FILE`. See step 6 in the guides below.
+
+## Client-Side
+
+*Assuming the pre-requisites are met, this migration should take ~1 developer day to complete.*
+
+
+1. Add the new SDK to your app
+  * Swift Package Manger
+      * In Xcode, add the PayPal SDK as a [package dependency](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) to your Xcode project. Enter https://github.com/paypal/iOS-SDK.git as the package URL. 
+      * Tick the `PayPalNativePayments`, `PaymentButtons`, and `CorePayments` boxes to add the libraries to your app.
+
+  * CocoaPods
+      * Include the `PayPalNativePayments`, and `PaymentButtons` sub-modules in your `Podfile`:
+        ```ruby
+        pod 'PayPal/PayPalNativePayments'
+        pod 'PayPal/PaymentButtons'
+        ```
+
+    *Note*: Updating your existing `paypalcheckout-ios` dependency to the latest will resolve any dependency conflicts.
```

Comment:
Should we add steps for how to update this?

Reply:
Eh, I didn't think that was this guide's responsibility. The top of this guide links to the NXO public docs

Reply:
That's fair!